src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx

```diff
@@ -189,4 +189,7 @@
   <data name="InvalidArgument" xml:space="preserve">
     <value>Invalid argument '{0}', did you mean:</value>
   </data>
+  <data name="WindowStyleArgumentNotImplemented" xml:space="preserve">
+    <value>WindowStyle is not implemented on this platform.</value>
```

Comment:
Maybe: "Parameter -WindowStyle is not implemented on this platform."

Reply:
Looks like the rest of the strings in the resource file refer the base as an argument and then parameter for anything the argument needs.

e.g.
-Argument
-Argument Parameter

Either way "WindowStyle is not implemented on this platform" should be prefixed with what ever is correct. Please advise.

Reply:
Should be `-Parameter <argument>`
See `Cannot process the command because the -InputFormat parameter requires an argument. Specify a valid format argument for this parameter.`

Reply:
PR updated. Thanks.